app/src/main/java/com/microsoft/office365/snippetapp/CalendarStories/GetFilteredImportantEvents.java

```diff
@@ -68,24 +67,18 @@ public String execute() {
             calendarSnippets.deleteCalendarEvent(testEvent.getId());
 
         } catch (ExecutionException | InterruptedException e) {
-            e.printStackTrace();
-            String formattedException = APIErrorMessageHelper.getErrorMessage(e.getMessage());
-            Log.e("EventFilter", formattedException);
-            return StoryResultFormatter.wrapResult(
-                    "Filter important events: " + formattedException
-                    , false
-            );
+            return BaseExceptionFormatter(e, STORY_DESCRIPTION);
         }
         if (isSucceeding) {
-            return StoryResultFormatter.wrapResult("FilterImportantEventsStory: Important events found.", true);
+            return StoryResultFormatter.wrapResult(STORY_DESCRIPTION + ": Important events found.", true);
         } else {
-            return StoryResultFormatter.wrapResult("FilterImportantEventsStory: Important events not found.", false);
+            return StoryResultFormatter.wrapResult(STORY_DESCRIPTION + ": Important events not found.", false);
         }
     }
 
     @Override
     public String getDescription() {
-        return "Gets events filtered by most important";
+        return STORY_DESCRIPTION;
```

Comment:
A few stories still return a text string. Can you make them all return STORY_DESCRIPTION for consistency?